src/main/resources/config/liquibase/master.xml

```diff
@@ -50,6 +50,7 @@
     <include file="classpath:config/liquibase/changelog/20230713171500_changelog.xml" relativeToChangelogFile="false"/>
     <include file="classpath:config/liquibase/changelog/20230721135400_changelog.xml" relativeToChangelogFile="false"/>
     <include file="classpath:config/liquibase/changelog/20230812135300_changelog.xml" relativeToChangelogFile="false"/>
+    <include file="classpath:config/liquibase/changelog/20230830183901_changelog.xml" relativeToChangelogFile="false"/>
```

Comment:
This should be at the end of the list of changes

Reply:
Right, I guess this is because of a merge commit. And are they supposed to be in chronological order, i.e., L54 should be before L50?
https://github.com/ls1intum/Artemis/blob/bc5fe9c2ea152f1b6e81c15775d3aa6e28943069/src/main/resources/config/liquibase/master.xml#L54